Aquarius/Src/Aquarius/Core/Application.cpp

```diff
@@ -1,7 +1,7 @@
-
 #include "Application.h"
-#include "Aquarius/Core/Log.h"
 #include "Aquarius/Core/Input.h"
```

Comment:
If the file is in the same sub-directory as the includes are we including the path or just the name? We have done this differently throughout.